code/src/index.css

```diff
@@ -1,13 +1,283 @@
+html {
+  background: url(./pink.jpeg);
+  background-position-x: center;
+  background-position-y: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed; 
+  background-size: cover;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  box-sizing: border-box;
+  font-family: 'Open Sans', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.container {
+  height: 100vh;
+  width: 320px; 
+  margin: 0 auto;
+  display: grid;
+  align-items: center;
+}
+
+.question-container {
+  padding: 22px 28px 140px 28px;
+  display: grid;
+  text-align: left;
+  background-color: rgba(230, 230, 230, 0.3);
```

Comment:
nice background for this box. A simple thing that makes it look really stylish ^^